src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_svala.cpp

```diff
@@ -366,22 +367,13 @@ class boss_svala : public CreatureScript
                             events.ScheduleEvent(EVENT_RITUAL_STRIKE, 2s, 0, SACRIFICING);
                             break;
                         case EVENT_RITUAL_STRIKE:
-                            me->StopMoving();
-                            me->GetMotionMaster()->MoveIdle();
-                            me->InterruptNonMeleeSpells(true);
                             DoCast(me, SPELL_RITUAL_STRIKE_TRIGGER, true);
-                            events.ScheduleEvent(EVENT_RITUAL_DISARM, 200ms, 0, SACRIFICING);
-                            break;
-                        case EVENT_RITUAL_DISARM:
-                            DoCast(me, SPELL_RITUAL_DISARM);
                             break;
                         case EVENT_FINISH_RITUAL:
                             me->SetDisableGravity(false, true);
-                            SetCombatMovement(true);
-
+                            me->SetReactState(REACT_AGGRESSIVE);
                             if (Unit* target = me->SelectNearestPlayer(100.0f))
                                 AttackStart(target);
```

Comment:
this should also be obsolete because UpdateVictim will handle care of the attacks

Reply:
thx ;)